app/aws/s3.py

```diff
@@ -466,23 +466,7 @@ def get_personalisation_from_s3(service_id, job_id, job_row_number):
 
     set_job_cache(job_cache, f"{job_id}_personalisation", extract_personalisation(job))
 
-    # If we can find the quick dictionary, use it
-    if job_cache.get(f"{job_id}_personalisation") is not None:
-        personalisation_to_return = job_cache.get(f"{job_id}_personalisation")[0].get(
-            job_row_number
-        )
-        if personalisation_to_return:
-            return personalisation_to_return
-        else:
-            current_app.logger.warning(
-                f"Was unable to retrieve personalisation from lookup dictionary for job {job_id}"
-            )
-            return {}
-    else:
-        current_app.logger.error(
-            f"Was unable to construct lookup dictionary for job {job_id}"
-        )
-        return {}
+    return job_cache.get(f"{job_id}_personalisation")[0].get(job_row_number)
```

Comment:
Just double checking that it's safe to remove the checks above from here.  I'm all for refactoring code and eliminating things we don't need, but just want to be sure that we know we can grab things from the dictionary here without it failing!

Reply:
Yes, safe to remove.  All this checking was actually due to the problem we had where sqlalchemy was setting the wrong created_at time and we were losing data pre-maturely.